Sources/Implementation/DefaultNotificationCenter.swift

```diff
@@ -205,8 +205,6 @@ extension DefaultNotificationCenter {
             if let event = try? JSONSerialization.jsonObject(with: eventData, options: []) as? [String: Any] {
                 let args: [Any] = [url, event]
                 self.sendNotifications(type: NotificationType.logEvent.rawValue, args: args)
-            } else {
```

Comment:
why removed this logging.

Reply:
It's the only debug message which may not be helpful. Not worth for adding a logger here.